src/_scss/pages/state/_contentWrap.scss

```diff
@@ -29,11 +29,20 @@
 
         .state-section__row {
             width: 100%;
-            margin-bottom: rem(50);
+            margin-bottom: rem(32);
 			@include media($tablet-screen) {
 				@include display(flex);
 			}
 
+			&.award-row {
+				margin-bottom: 33px;
```

Comment:
this should be rem(32); marcy said in the design review that the 33 is an error in zeplin